test/test_interactive.py

```diff
@@ -306,6 +306,34 @@ def test_audio_worklet_tone_generator(self):
   def test_audio_worklet_modularize(self):
     self.btest('webaudio/audioworklet.c', expected='0', args=['-sAUDIO_WORKLET', '-sWASM_WORKERS', '-sMINIMAL_RUNTIME', '-sMODULARIZE'])
 
+  # Tests an AudioWorklet with multiple stereo inputs mixing in the processor to a single stereo output (4kB stack)
+  def test_audio_worklet_stereo_io(self):
+    os.mkdir('audio_files')
+    shutil.copy(test_file('webaudio/audio_files/emscripten-beat.mp3'), 'audio_files/')
+    shutil.copy(test_file('webaudio/audio_files/emscripten-bass.mp3'), 'audio_files/')
+    self.btest('webaudio/audioworklet_in_out_stereo.c', expected='0', args=['-sAUDIO_WORKLET', '-sWASM_WORKERS'])
+
+  # Tests an AudioWorklet with multiple stereo inputs copying in the processor to multiple stereo outputs (6kB stack)
+  def test_audio_worklet_2x_stereo_io(self):
+    os.mkdir('audio_files')
+    shutil.copy(test_file('webaudio/audio_files/emscripten-beat.mp3'), 'audio_files/')
+    shutil.copy(test_file('webaudio/audio_files/emscripten-bass.mp3'), 'audio_files/')
+    self.btest('webaudio/audioworklet_2x_in_out_stereo.c', expected='0', args=['-sAUDIO_WORKLET', '-sWASM_WORKERS'])
+
+  # Tests an AudioWorklet with multiple mono inputs mixing in the processor to a single mono output (2kB stack)
+  def test_audio_worklet_mono_io(self):
+    os.mkdir('audio_files')
+    shutil.copy(test_file('webaudio/audio_files/emscripten-beat-mono.mp3'), 'audio_files/')
+    shutil.copy(test_file('webaudio/audio_files/emscripten-bass-mono.mp3'), 'audio_files/')
+    self.btest('webaudio/audioworklet_in_out_mono.c', expected='0', args=['-sAUDIO_WORKLET', '-sWASM_WORKERS'])
+
+  # Tests an AudioWorklet with multiple mono inputs copying in the processor to L+R stereo outputs (3kB stack)
+  def test_audio_worklet_2x_hard_pan_io(self):
+    os.mkdir('audio_files')
+    shutil.copy(test_file('webaudio/audio_files/emscripten-beat-mono.mp3'), 'audio_files/')
+    shutil.copy(test_file('webaudio/audio_files/emscripten-bass-mono.mp3'), 'audio_files/')
+    self.btest('webaudio/audioworklet_2x_in_hard_pan.c', expected='0', args=['-sAUDIO_WORKLET', '-sWASM_WORKERS'])
```

Comment:
Can you use `btest_exit` and remove the `expected` arg?

Reply:
Question here: I wrote the tests so they do use the exit by verifying playback happened. You'd like me to remove the verifications in the tests too?

Reply:
No, I'm just suggesting that if you are using the exit code as the result and the result is zero then you can use `btest_exit` which by default checks for zero exit code.

Having said that, I think calling exit() from a audio worklet currently doesn't' work because proxying doesn't work, so maybe its not easy in this case?

I've been trying to move from `btest` to `btest_exit` whereever we can.

Reply:
Ah, makes sense. I'll can take a look next week (I'm done for a few days now).

Reply:
I changed to `btest_exit` and `emscripten_force_exit()`. The test instantly exits.

So I build with `-sNO_EXIT_RUNTIME` and now it doesn't exit at all. I can see the reported result if I switch back to `btest`, so `report_result?exit:0`, but it continues running.

Is this the behaviour you'd expect when you say the proxying doesn't work?

Reply:
You need to add `exit_with_live_runtime()` (or better, `emscripten_runtime_keeplive_push()`) at the end of your `main` function, otherwise returning from `main` signals the end of the program.

Reply:
See https://github.com/emscripten-core/emscripten/pull/22776

Reply:
The tests are all migrated to `btest_exit()`